src/bioregistry/data/bioregistry.json

```diff
@@ -56776,7 +56776,8 @@
       "name": "InterPro ID",
       "pattern": "^IPR\\d{6}$",
       "prefix": "P2926",
-      "uri_format": "http://www.ebi.ac.uk/interpro/entry/$1"
+      "uri_format": "https://www.ebi.ac.uk/interpro/entry/InterPro/$1"
+      "uri_format_rdf": "http://www.ebi.ac.uk/interpro/entry/$1"
```

Comment:
These changes are still under `wikidata`. We shouldn't change the content within mapped keys like `wikidata`, only outside.